app/models/document_upload.rb

```diff
@@ -0,0 +1,71 @@
+class DocumentUpload
+  include Virtus.model
+
+  attribute :tempfile, Object
+  attribute :content_type, String
+  attribute :original_filename, String
+
+  attr_accessor :errors
+
+  # TODO: decide on the final allowed max size and content types
+  #
```

Comment:
I agree that we need to have a proper discussion about the filetype and size limits. In the meantime, please could we add text/csv and the common spreadsheet types?